app/src/main/java/br/com/catbag/gifreduxsample/ui/components/FeedComponent.java

```diff
@@ -58,29 +57,33 @@ public FeedComponent(Context context, AttributeSet attrs, int defStyleAttr) {
 
     @Override
     public void view() {
-        final List<Gif> gifs = mGifs;
         Recycler.view(() -> {
             if (mLayoutManager == null) {
                 mLayoutManager = new LinearLayoutManager(getContext(),
                         LinearLayoutManager.VERTICAL, false);
             }
             Recycler.layoutManager(mLayoutManager);
```

Comment:
Acredito que deve ser chamado sempre que a view é renderizada virtualmente. Por que uma chamada ao método view não necessariamente vai redesenhar a view na tela. Caso ela esteja dentro do if, nas próximas vezes que passar no método view, e a Recycler for recriada virtualmente, ela não vai conter um layout manager nem o fixedSize true.